install_config/storage_examples/privileged_pod_storage.adoc

```diff
@@ -8,110 +8,104 @@
-link:../persistent_storage/persistent_storage_glusterfs.html#gfs-creating-persistent-volume[persistent volume],
-link:../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent volume claim], and a link:#create-priv-pod[privileged pod].
-** For this guide, these will be:
-*** `gluster-endpoints-service.yaml`
-*** `gluster-endpoints.yaml`
-*** `gluster-pvc.yaml`
-*** `gluster-pv.yaml`
-*** `gluster-nginx-pod.yaml`
+== Prerequisites
+* An existing Gluster volume. 
+* *glusterfs-fuse* installed on all hosts.
+* Definitions for GlusterFS:
+** link:../persistent_storage/persistent_storage_glusterfs.html#creating-gluster-endpoints[Endpoints and services]: *_gluster-endpoints-service.yaml_* and *_gluster-endpoints.yaml_*
+** link:../persistent_storage/persistent_storage_glusterfs.html#gfs-creating-persistent-volume[Persistent volumes]: *_gluster-pv.yaml_*
+** link:../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent volume claims]: *_gluster-pvc.yaml_*
+** link:#create-priv-pvc[Privileged pods]: *_gluster-nginx-pod.yaml_*
 * A user with the
-link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[*cluster-admin*] role binding.
-** For this guide, that user is called `admin`
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[*cluster-admin*] role binding. For this guide, that user is called `admin`.
 
 
-== Make the Volume Accessible
+== Making the Volume Accessible
 
 
 [[create-priv-pv]]
-=== Create the Persistent Volume
+=== Creating the Persistent Volume
 
-Creating the *PersistentVolume* will make the storage accessible to users,
+Creating the *PersistentVolume* makes the storage accessible to users,
 regardless of projects.
 
-.*As admin*
-[source]
+. As the admin, create the service, endpoint object, and persistent volume:
++
 ----
 $ oc create -f gluster-endpoints-service.yaml
 $ oc create -f gluster-endpoints.yaml
 $ oc create -f gluster-pv.yaml
 ----
 
-Verify Success
-
-[source]
-----
-$ oc get pv
-NAME                     LABELS    CAPACITY   ACCESSMODES   STATUS      CLAIM     REASON    AGE
-gluster-default-volume   <none>    2Gi        RWX           Available                       2d
-----
+. Verify that the objects were created:
++
 ----
 $ oc get svc
 NAME              CLUSTER_IP      EXTERNAL_IP   PORT(S)   SELECTOR   AGE
 gluster-cluster   172.30.151.58   <none>        1/TCP     <none>     24s
 ----
++
 ----
 $ oc get ep
 NAME              ENDPOINTS                           AGE
 gluster-cluster   192.168.59.102:1,192.168.59.103:1   2m
 ----
++
+----
+$ oc get pv
+NAME                     LABELS    CAPACITY   ACCESSMODES   STATUS      CLAIM     REASON    AGE
+gluster-default-volume   <none>    2Gi        RWX           Available                       2d
+----
 
 [[create-regular-user]]
-== Create a Regular User
+== Creating a Regular User
```

Comment:
@bfallonf I'm confused by the way the TOC headings are jumping up and down (mostly by "Creating the PVC" being a subsection of "Creating a Regular User"). Wondering if makes more sense to either have all the "Creating"s be under "Making the Volume Accessible", i.e.:

```
== Making the Volume Accessible
=== Creating the Persistent Volume
=== Creating a Regular User
=== Creating the Persistent Volume Claim
```

Or just drop "Making the Volume Accessible" (since it's just a heading w/ no content anyway atm), and let the "Creating"s just be level 1 headings themselves, i.e.:

```
== Creating the Persistent Volume
== Creating a Regular User
== Creating the Persistent Volume Claim
```